index.js

```diff
@@ -354,10 +371,35 @@ const runUtilForAllInDir = () => {
   });
 };
 
+/**
+ * Tests if a file exists
+ * @param  string file The path to the file being read
+ * @return boolean Boolean indicating if the file exists
+ */
+function fileExists(file) {
+  try {
+    fs.accessSync(file, fs.constants.R_OK | fs.constants.W_OK);
```

Comment:
Sorry for the delay. Had a busy week with family and stuff. Anyway, I'm thinking we support node v <= 4.6. Any thoughts?